roles/kube-node-common/tasks/kubelet.yml

```diff
@@ -78,10 +78,20 @@
     state: present
     update_cache: yes
 
+- name: Download cri-tools
+  get_url:
+    url: https://github.com/kubernetes-sigs/cri-tools/releases/download/v{{ critools_version }}/{{ item }}-v{{ critools_version }}-linux-amd64.tar.gz
+    dest: "/tmp/{{ item }}-v{{ critools_version }}-linux-amd64.tar.gz"
+    mode: 0600
+    checksum: "sha256:{{ lookup('vars', 'critools_sha_' + item + '_v' + critools_version)}}"
```

Comment:
I am fine with the proposed solution, but why not just use a map to avoid the lookup('vars', ...)?

Feel free to mark it as solved if you want to stick to the existing implementation.

Reply:
Uhmmmm ok. Looks easier to understand. So i will change it ;)

Reply:
Done!